### PR TITLE
Capitalize first_name instead of upcase in NotificationEmail

### PR DESCRIPTION
**Pull request by Thrillberg**
## Summary
This PR changes the email we send from SimpleForms such that first names are capitalized (`John`) instead of upcased (`JOHN`).

### CODE REVIEW

**Review by jeana-adhoc — DISMISSED** [review]
LGTM! Thanks for getting this in.

**Review by pennja — APPROVED** [review]
Looks good to me! 👍

**Comment by stevenjcumming** [comment]
@Thrillberg if someone has two first names (double name), the second name won't be capitalized. Consider `titleize` if a double name is possible

https://apidock.com/rails/String/titleize

**Review by stevenjcumming — DISMISSED** [review]

File: modules/simple_forms_api/spec/services/notification_email_spec.rb [hunk 1]
```diff
@@ -497,7 +497,7 @@
             user.va_profile_email,
             'form21_0966_confirmation_email_template_id',
             {
-              'first_name' => user.first_name.upcase,
+              'first_name' => user.first_name.capitalize,
```
Comment (rmtolmach) [inline modules/simple_forms_api/spec/services/notification_email_spec.rb:500]:
```suggestion
              'first_name' => user.first_name.titleize,
```

File: modules/simple_forms_api/spec/services/notification_email_spec.rb [hunk 2]
```diff
@@ -151,7 +151,7 @@
                 email,
                 "form21_10210_#{notification_type}_email_template_id",
                 {
-                  'first_name' => 'JOHN',
+                  'first_name' => 'John',
```
Comment (rmtolmach) [inline modules/simple_forms_api/spec/services/notification_email_spec.rb:154]: Can you change at least one of these first names in a spec to be a double first name? `'John Luke'`?

Comment (Thrillberg) [reply modules/simple_forms_api/spec/services/notification_email_spec.rb:154]: Yep! I added one elsewhere. Thanks!

**Review by pennja — APPROVED** [review]
Looks good! 👍

**Comment by Thrillberg** [comment]
[Use this PR instead](https://github.com/department-of-veterans-affairs/vets-api/pull/18894). Closing.

**Pull request closed without merging**